examples/amp-story/reactions.html

```diff
@@ -48,12 +48,12 @@ <h2>What is the best story ever?</h2>
                 id="first-cup-dark"
                 endpoint="http://localhost:3000/reactions"
                 theme="dark"
-                chip-corner="sharp">
-              <h1>Who won the first soccer world cup?</h1>
-              <option>France</option>
-              <option>Spain</option>
-              <option correct>Uruguay</option>
-              <option>Brazil</option>
+                chip-corner="sharp"
+                prompt-text="Who won the first soccer world cup?"
+                option-1-text="France"
+                option-2-text="Spain"
+                option-3-text="Uruguay" option-3-correct="correct"
```

Comment:
Should `option-3-correct="correct"` be a boolean? Or do you have use cases in mind that'd require this to be a string?

Reply:
I was thinking that we could enforce the options to always have the "correct" value, but we could check for the flag existing, and not worry about the value at all. When parsing the attributes, containing or not the flag is what matters in the end, as the .js file checks option["correct"], so the flag could have any string, or no string at all. We could validate option-3-correct without any value, or force it to be whatever string we want ("correct" in this case).

Since it improves readability, I made these attributes value-less, since the value doesn't really change anything.